platform/feature-api/api/src/pod-api/mock-pod.ts

```diff
@@ -0,0 +1,10 @@
+import { DefaultPod } from "./default";
+import { dataset } from "@rdfjs/dataset";
+import { Volume } from "memfs";
+import { FS } from "./fs";
+
+export class MockPod extends DefaultPod {
```

Comment:
Why not just rename `DefaultPod` to `MockPod`? I really don't see what need we have for `DefaultPod` - it's just a constant source of confusion. But if there's any entanglements that make this difficult, I'm OK with eliminating it later.

Reply:
Main reason is I don't want to pull all these dependencies as production dependencies. Also, right now there's not a lot of difference, but in order to implement some other `poly*` I think it's better to do it in this Mock, and not in the Default one.
An alternative is, of course, to spin off DefaultPod to its own file and make it MockPod directly, and then use it only for tests, but that would need another degree of refactoring.